frontend/packages/core/src/Contexts/storage-context/helpers.tsx

```diff
@@ -0,0 +1,58 @@
+import type { clutch as IClutch } from "@clutch-sh/api";
+
+import type { HydratedData, StorageState } from "./types";
+
+// API data comes in as an array, this will rotate it into an object usable by the StorageContext
+const rotateDataFromAPI = (data: IClutch.shortlink.v1.IShareableState[]): HydratedData => {
+  const hydrated: HydratedData = {};
+
+  data.forEach(({ key = "", state = {} }) => {
```

Comment:
why the default value for key?

Reply:
Think I was just defaulting it all, I'll adjust